### ISSUE TITLE
Global shortcut generates error when used with accelerator `num0`

### ISSUE THREAD

**Issue by AhmedLoud** [electron/electron#16206]
Hi, I have an error when trying to register the numpad 0 accelerator using global shortcut

I am running electron version 4.0.
**Expected Behavior**
Global shortcut call back triggers when hitting numpad 0

**Actual behavior**
My app does not even start generating an error:
`TypeError: Error processing argument at index 0, conversion failure from num0`

**To Reproduce**
I created a sample app that reproduce the issue: https://github.com/AhmedLoud/RegisterNumPadShortcutElectronIssue

I will appreciate some help from you to tell me what am I doing wrong here.

Thank you

**Comment by codebytere** [comment]
@AhmedLoud shortcuts are required to have modifiers; see [accelerator documentation](https://electronjs.org/docs/api/accelerator)

> Accelerators are Strings that can contain multiple modifiers and a single key code, combined by the + character, and are used to define keyboard shortcuts throughout your application.

if you change that to `ctrl+num0` you'll see that it works.

**Comment by AhmedLoud** [comment]
@codebytere I tried to change it to ctrl+num0 and I still get the same error: [https://github.com/AhmedLoud/RegisterNumPadShortcutElectronIssue](https://github.com/AhmedLoud/RegisterNumPadShortcutElectronIssue)

More over I noticed that almost all the regular keys works without any modifier. I added an example with the key `A` in the sample app.

Would it be possible to have the same behavior for the numpad as well (no modifiers required)?
